types/ospath/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for ospath 1.2
+// Project: https://github.com/jprichardson/ospath#readme
+// Definitions by: Kishan Gajera <https://github.com/kgajera>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export const __platform: string;
```

Comment:
this is internal detail, do you need to export this?

Reply:
Removed